frontend/public/components/storage-class.tsx

```diff
@@ -12,29 +12,34 @@ const { common } = Kebab.factory;
 const menuActions = [...common];
 
 const defaultClassAnnotation = 'storageclass.kubernetes.io/is-default-class';
-const isDefaultClass = (storageClass: K8sResourceKind) => _.get(storageClass, ['metadata', 'annotations', defaultClassAnnotation], 'false');
+const isDefaultClass = (storageClass: K8sResourceKind) => {
+  const BoolValue = _.get(storageClass, ['metadata', 'annotations', defaultClassAnnotation], 'false');
+  return ( BoolValue === 'true' );
+};
 
 const StorageClassHeader = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="provisioner">Provisioner</ColHead>
+  <ColHead {...props} className="col-sm-5 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-5 col-xs-6" sortField="provisioner">Provisioner</ColHead>
   <ColHead {...props} className="col-sm-2 hidden-xs" sortField="reclaimPolicy">Reclaim <span className="hidden-sm">Policy</span></ColHead>
-  <ColHead {...props} className="col-sm-2 hidden-xs" sortField={`metadata.annotations['${defaultClassAnnotation}']`}>Default <span className="hidden-sm">Class</span></ColHead>
 </ListHeader>;
 
 const StorageClassRow: React.SFC<StorageClassRowProps> = ({obj}) => {
   return <div className="row co-resource-list__item">
-    <div className="col-sm-4 col-xs-6 co-break-word">
-      <ResourceLink kind={StorageClassReference} name={obj.metadata.name} namespace={undefined} title={obj.metadata.name} />
+    <div className="col-sm-5 col-xs-6 co-break-word">
+      <ResourceLink
+        inline={true}
+        kind={StorageClassReference}
+        name={obj.metadata.name}
+        namespace={undefined}
+        title={obj.metadata.name} />
+      { isDefaultClass(obj) ? <i><small className="text-muted"> &ndash; &nbsp; &nbsp; Default </small></i> : null }
```

Comment:
We use small/text-muted without italics in other places in console. We should try to be consistent.

```suggestion
      { isDefaultClass(obj) && <small className="text-muted"> &ndash; &nbsp; &nbsp; Default </small> }
```

Reply:
It would be better to assign it a custom class and control the margin spacing like... 
`<span class="text-muted small storage-default">– Default </span>`
```
.storage-default {
    margin-left: 5px;
    white-space: nowrap;
}
```

<img width="1110" alt="Screen Shot 2019-03-26 at 3 17 13 PM" src="https://user-images.githubusercontent.com/1874151/55027068-c2223f80-4fda-11e9-854f-0b69f8dfec3d.png">

Reply:
Prefer `co-storage-default` to use our prefix

Reply:
```html
<span className="small text-muted co-storage-default">&ndash; Default</span>
```

Reply:
Ack. Thanks :)